src/test/test-execute.c

```diff
@@ -1074,6 +1075,11 @@ static void test_exec_ambientcapabilities(Manager *m) {
 static void test_exec_privatenetwork(Manager *m) {
         int r;
 
+        if (!have_net_dummy) {
+                log_notice("Skipping %s, dummy network interface not available", __func__);
```

Comment:
`return (void) log_notice(`

Reply:
Changed - I saw both code styles existed, but in this very function, it was not used that way.